src/main/java/core/net/OnlineWorker.java

```diff
@@ -302,19 +328,27 @@ public static boolean downloadMatchData(MatchKurzInfo info, boolean refresh) {
 							}
 						}
 
-						// get the other team
-						int otherId;
-						if (info.isHomeMatch()) {
-							otherId = info.getGuestTeamID();
-						} else {
-							otherId = info.getHomeTeamID();
-						}
-						if (otherId > 0) {
-							Map<String, String> otherTeam = getTeam(otherId);
-							info.setIsDerby(getRegionId(otherTeam) == HOVerwaltung.instance().getModel().getBasics().getRegionId());
-							info.setIsNeutral(info.getArenaId() != HOVerwaltung.instance().getModel().getStadium().getArenaId()
-									&& info.getArenaId() != getArenaId(otherTeam));
-							downloadTeamLogo(otherTeam);
+						if (info.getHomeTeamID() > 0) {
```

Comment:
`> 0` means when set, right?
I am wondering why the default value it `-1`. So why not `> -1` or `!= -1`?

Reply:
hattrick gives negative ids to "verlegenheitstruppen" which are the opponents in first cup rounds.

Reply:
Sorry, I don't get it.
"verlegenheitstruppen" is when you are for free automatically in the next round in the cup (first round only) ?

Reply:
nearly for free - these teams are very weak and created by the hattrick engine.